src/features/autofraction.ts

```diff
@@ -105,8 +105,13 @@ export const runAutoFractionCursor = (view: EditorView, ctx: Context, range: Sel
 			numerator = numerator.slice(1, -1);
 		}
 	}
-
-	const replacement = `${settings.autofractionSymbol}{${numerator}}{$0}$1`
+	
+	let replacement: string;
+	if (numerator === "()") {
+		replacement = `${settings.autofractionSymbol}{$0}{$1}$2`;
+	} else {
+		replacement = `${settings.autofractionSymbol}{${numerator}}{$0}$1`;
+	}
```

Comment:
The ternary operator can be used here for conciseness:

```suggestion
	
	const replacement = numerator === "()" ? `${settings.autofractionSymbol}{$0}{$1}$2` : `${settings.autofractionSymbol}{${numerator}}{$0}$1`;
```

Reply:
Thought about that and the snippet below
```ts
numerator = numerator || "$0"
const replacement = `${settings.autofractionSymbol}{${numerator}}{$1}$2
```
but wasn't sure if I wasn't "complicating" things further.

numerator should be checked against an empty string instead of `()`. 
I somehow didn't catch that when I went from testing my code to finalizing it